docs/standard/serialization/system-text-json-preserve-references.md

```diff
@@ -39,6 +39,29 @@ For more information about how references are serialized and deserialized, see <
 
 The <xref:System.Text.Json.Serialization.ReferenceResolver> class defines the behavior of preserving references on serialization and deserialization. Create a derived class to specify custom behavior. For an example, see [GuidReferenceResolver](https://github.com/dotnet/docs/blob/9d5e88edbd7f12be463775ffebbf07ac8415fe18/docs/standard/serialization/snippets/system-text-json-how-to-5-0/csharp/GuidReferenceResolverExample.cs).
 
+## Multiple Serialize calls
```

Comment:
```suggestion
## Persisting reference metadata across multiple serialization and deserialization calls
```

I'd generalize this a bit since the same pattern can be used for deserialization, with some tweaks.